src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -37,13 +37,17 @@
      {:color :blue})
    :render
    (fn [{:keys [props]}]
-     [:span {:style {:display "inline-block"
-                     :backgroundColor (case (:color props)
-                                        (:button-blue colors))
-                     :color "white" :fontWeight 500
-                     :borderRadius 2 :padding "0.7em 1em"
-                     :cursor "pointer"}
-             :onClick (fn [e] ((:onClick props) e))}
+     [:a {:style {:display "inline-block"
+                  :backgroundColor (case (:color props)
+                                     (:button-blue colors))
+                  :color "white" :fontWeight 500
+                  :borderRadius 2 :padding "0.7em 1em"
+                  :textDecoration "none"}
+          :href "javascript:;"
```

Comment:
Just FYI: I pretty much add this {:href "javascript:;"} boilerplate to every single :a element in my own codebases. When you're thinking about factoring out common functionality, you might consider this case.